providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/strategy/ResolveFactory.java

```diff
@@ -0,0 +1,12 @@
+package dev.openfeature.contrib.providers.flagd.strategy;
```

Comment:
Minor - this can also fit into a `telemetry` package

Reply:
I didn't want to change the behavior with this PR, but we should always provide an instrumented version IMHO.
If in the App code, the SDK is not registered, then it will be a no-op implementation.

Reply:
Added to https://github.com/open-feature/java-sdk-contrib/issues/390